ansys/dpf/core/dpf_operator.py

```diff
@@ -144,14 +163,16 @@ def connect(self, pin, inpt, pin_out=0):
         request = operator_pb2.UpdateRequest()
         request.op.CopyFrom(self._message)
         request.pin = pin
-        _fillConnectionRequestMessage(request, inpt, pin_out)
+        tmp = _fillConnectionRequestMessage(request, inpt, self._server, pin_out)
         if inpt is self:
             raise ValueError("Cannot connect to itself.")
         self._stub.Update(request)
 
     @protect_grpc
     def get_output(self, pin=0, output_type=None):
         """Retrieve the output of the operator on the pin number.
```

Comment:
Short and long description should have a line between.
```suggestion
        """Retrieve the output of the operator on the pin number.

```